docs/backend-system/architecture/02-backends.md

```diff
@@ -17,24 +17,27 @@ Below is a simple example of a backend that installs only the catalog plugin and
 
 ```ts
 import { createBackend } from '@backstage/backend-defaults';
-import { catalogPlugin } from '@backstage/plugin-catalog-backend';
+import scaffolderPlugin from '@backstage/plugin-scaffolder-backend';
 
 // Create your backend instance
 const backend = createBackend();
 
-// Install all desired features
-backend.add(catalogPlugin());
+// Install desired features
+backend.add(import('@backstage/plugin-catalog-backend'));
+
+// Features can also be installed using an explicit reference
+backend.add(scaffolderPlugin());
```

Comment:
Is there a difference in build optimization between the two import methods?

Reply:
I don't think so tbh. And at runtime they're both sync right now, and in the future they'll both be async afaik